test/channel/base_test.rb

```diff
@@ -92,6 +111,23 @@ def rm_rf
     assert_nil @channel.last_action
   end
 
+  test "try calling a public method defined on Base" do
+    @channel.perform_action 'action' => :kick
+    assert_nil @channel.last_action
```

Comment:
Yes, that's right. I used the same testing pattern used by @lifo [here](https://github.com/cristianbica/actioncable/blob/channel-action-dispatched/test/channel/base_test.rb#L109)